src/control/system/database_test.go

```diff
@@ -135,97 +129,18 @@ func TestSystem_Database_filterMembers(t *testing.T) {
 	}
 }
 
-func TestSystem_Database_checkReplica(t *testing.T) {
```

Comment:
Adding unit tests for the new IsReplica/IsLeader/etc. functions?

Reply:
Yeah, will do.